docusaurus/docs/iOS/customization/theming.md

```diff
@@ -1,3 +1,72 @@
 ---
 title: Theming
----
\ No newline at end of file
+---
+
+You can customize the look and feel of all UI components provided by `StreamChatUI`. The SDK allows you to change appearance of components such as colors and fonts via the `Appearance` component. Changes to appearance should be done as early as possible in your application, `SceneDelegate` and `AppDelegate` are usually the right place to do this. The SDK comes with a singleton object `Appearance.default`.
```

Comment:
> The SDK comes with a singleton object `Appearance.default`. 

This is true. Just a note that customers can also create custom `Appearance` instance, then set the properties to it and then assign it `Appearance.default =  myApperance` or basically any `AppearanceProvider` implementing view so those it can have different appearance than rest of SDK.  This is very specific usecase but I think it's still worth mentioning.

Reply:
I actually think a better name for this is `Appearance.current`. This pattern is also user in other Apple's APIs. `default` is a bit confusing, it looks that there's more appearance objects, where in fact we can only have one. Either `current` or `shared` are better options, IMO.

Reply:
`Appearance` and `Components` are based on the responder chain. You can have different objects for different objects. In the future, we will have built-in presets for specific UI: like messaging (the current default), live chat, etc.

Reply:
`.default` means - if you don't specify your custom appearance, we will use the default one.